packages/ui/src/working-groups/constants/index.ts

```diff
@@ -8,7 +8,7 @@ export const GroupIdToGroupParam = {
   membershipWorkingGroup: 'Membership',
   distributionWorkingGroup: 'Distribution',
   storageWorkingGroup: 'Storage',
-  operationsWorkingGroupAlpha: 'OperationsAlpha',
-  operationsWorkingGroupBeta: 'OperationsBeta',
-  operationsWorkingGroupGamma: 'OperationsGamma',
+  operationsWorkingGroupAlpha: 'Builders',
+  operationsWorkingGroupBeta: 'HR',
+  operationsWorkingGroupGamma: 'Marketing',
```

Comment:
I don't think this is right: This mapping is mostly used to define [`getWorkingGroupParam`](https://github.com/Joystream/pioneer/blob/dev/packages/ui/src/proposals/modals/AddNewProposal/getSpecificParameters.ts#L15) which is passed to the create proposal extrinsic, and AFAIK the chain doesn't know about the Builders, HR, and Marketing aliases.